File: programme/sum-of-digits/SumOfDigits.java

// Write a program to print sum of digits.
import java.util.Scanner;

class SumOfDigits {
    public static void main(String[] args) {
        int num, d, sum = 0;
        try (Scanner sc = new Scanner(System.in)) {
            System.out.print("Enter a number : ");
            num = sc.nextInt();
        }
        while (num != 0) {
            d = num % 10;
            sum = sum + d;
            num = num / 10;
        }
        System.out.println("Sum of the digits : " + sum);
    }
}

File: programme/sum-of-digits/sum-of-digits.js
/* 
Write a programmeto print sum of digits.
*/

function sumDigits(nums) {
  const arr = [...`${nums}`];
  return arr.reduce((acc, curr) => acc + Number(curr), 0);
}

const digits = 2332;
console.log("Input  :", digits);
console.log("Output :", sumDigits(digits));
